00_setup_and_funs.R

```diff
@@ -135,4 +135,9 @@ init_outputs_national <- function(purpose, geography){
     assign("rf_all", readRDS(file.path(path_outputs_national, purpose, geography, "rf_all.Rds")), envir = .GlobalEnv)
     assign("rq_all", readRDS(file.path(path_outputs_national, purpose, geography, "rq_all.Rds")), envir = .GlobalEnv)
   }
+  if(purpose == "school") {
+    assign("d_all", readRDS(file.path(path_outputs_national, purpose, geography,  "d_all.Rds")), envir = .GlobalEnv)
```

Comment:
Non-standard way of assigning objects. Sure it works but from a readibility perspective I'm not a fan. See that's how objects are created in the previous lines so this is a wider issue that does not affect this PR. Any particular reason why it's like that? I recall agreeing to refactor some of these and thought I did - you happy for me to re-commence that after this PR is done?

Reply:
this code was written by Ali.  my understanding was that you can't just use the "d_all <- readRDS..." normal code because this is putting something in the global environment, rather than just the local environment.  But as long as you maintain the global-environment functionality then certainly is is fine by me forthe code to be re-factored – as you say, if you make the change in one place it will be nice to do that consistently